examples/case_studies/conditional-autoregressive-model.myst.md

```diff
@@ -10,6 +10,19 @@ kernelspec:
   name: python3
 ---
 
+(conditional_autoregressive_model)=
+(about_CAR_models_in_pymc)=
+
+# About Conditional Autoregressive models in PyMC
+
+:::{post} Aug 14, 2020 
+:tags: spatial, autoregressive, count data
+:category: advanced, reference
```

Comment:
I think the notebook is more [explanation](https://diataxis.fr/explanation/) type than [reference](https://diataxis.fr/reference/).